packages/react/src/components/Persona/Persona.base.tsx

```diff
@@ -82,21 +83,23 @@ export const PersonaBase: React.FunctionComponent<IPersonaProps> = React.forward
      * to make it independent of the type of text passed
      * @param text - text to render
      */
-    const onRenderText = (text: string | undefined): IRenderFunction<IPersonaProps> | undefined => {
+    const onRenderText = (text: string | undefined, tooltip = true): IRenderFunction<IPersonaProps> | undefined => {
       // return default render behavior for valid text or undefined
       return text
-        ? (): JSX.Element => {
-            // default onRender behavior
-            return (
-              <TooltipHost
-                content={text}
-                overflowMode={TooltipOverflowMode.Parent}
-                directionalHint={DirectionalHint.topLeftEdge}
-              >
-                {text}
-              </TooltipHost>
-            );
-          }
+        ? tooltip
+          ? (): JSX.Element => {
+              // default onRender behavior
+              return (
+                <TooltipHost
+                  content={text}
+                  overflowMode={TooltipOverflowMode.Parent}
+                  directionalHint={DirectionalHint.topLeftEdge}
+                >
+                  {text}
+                </TooltipHost>
+              );
+            }
+          : (): JSX.Element => <>{text}</>
```

Comment:
If you remove the explicit return type you should be able to just return the text, rather than using a wrapping fragment